README.md

```diff
@@ -1,2 +1,31 @@
-# My-Portfolio-pj_microverse
-My first portfolio website built with HTML/CSS and JavaScript
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio Website
+
+This project is a Website version of my portfolio created for job searching as a software developer.
+
+## Built With
+
+- HTML/CSS and JavaScript
+
+## Authors
+
+👤 Karla Delgado
+
+- GitHub: [@githubhandle](https://github.com/karlavdelgadof)
+- Twitter: [@twitterhandle](https://twitter.com/karlavdelgadof)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/karla-delgado-613a32239/)
+
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse-Student-Project-1 Figma Templates
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please I would recommend you fix this broken link by including an MIT file that is customized to your project details to make your project look more professional.

Reply:
@karlavdelgadof Thanks for your feedback. Please could you explain to me how to add this MIT file? :pray: